pkg/kv/kvserver/kvserverbase/raftversion.go

```diff
@@ -68,26 +68,30 @@ const (
 	// bytes after the prefix represent the kvserverpb.RaftCommand.
 	//
 	RaftCommandPrefixLen = 1 + RaftCommandIDLen
+	// RaftVersionStandardPrefixByte is the first byte of a raftpb.Entry's
+	// Data slice for an Entry of encoding RaftVersionStandard.
+	RaftVersionStandardPrefixByte = byte(0)
+	// RaftVersionSideloadedPrefixByte is the first byte of a raftpb.Entry's Data
+	// slice for an Entry of encoding RaftVersionSideloaded.
+	RaftVersionSideloadedPrefixByte = byte(1)
```

Comment:
These could be type-safe too, but since the intention is to hide them, maybe not necessary. Will see after the next PR.

Reply:
I pondered that, but it turned out that because RaftCommandEncoding has its own type, all the uses that became "wrong" when I introduced the prefix bytes were red-squiggly already. But it might still make sense to introduce a type here down the line, we'll see.